rust/stackable-cockpit/src/platform/credentials.rs

```diff
@@ -11,7 +11,7 @@ pub type Result<T, E = Error> = std::result::Result<T, E>;
 #[derive(Debug, Snafu)]
 pub enum Error {
     #[snafu(display("failed to fetch data from Kubernetes API"))]
-    KubeClientFetch { source: k8s::Error },
+    KubeClientFetch { source: Box<k8s::Error> },
```

Comment:
nit: If you add `#[snafu(source(from(k8s::Error, Box::new)))]` to the `source` field it just works (tm) without the need to `map_err` when using `context`.
Applies to other snafu errors as well